### DISCUSSION TITLE
Majora's mask link gives fierce deity sword repeatedly

### DISCUSSION THREAD

**Discussion by ihigg** [jordanbtucker/totk-no-loot-boxes#21] [Q&A] [answered]
Using the Majora's mask link amiibo, I got the following rewards (in this order):

1. Majora's mask fabric
2. Fierce deity sword
3. Fierce deity armor

Afterwards, every time I use the amiibo I receive the fierce deity sword. I have been unable to get the fierce deity boots or mask.

EDIT: I got the boots after a few more scans. It would be nice if the clothing was easier to get.

**Comment by jordanbtucker** [comment, marked as answer]
With v2.0.1, once you've received the fabric, you'll receive the rest of the items with equal probability. So, the armor pieces are not any harder or easier to get than the sword. It's just luck.
